server/src/main/java/org/elasticsearch/plugins/PluginsService.java

```diff
@@ -205,8 +166,8 @@ private static void logPluginInfo(final List<PluginInfo> pluginInfos, final Stri
      * @return A stream of results
      * @param <T> The generic type of the result
      */
-    public <T> Stream<T> map(Function<Plugin, T> function) {
-        return plugins.stream().map(LoadedPlugin::instance).map(function);
+    public final <T> Stream<T> map(Function<Plugin, T> function) {
+        return this.plugins().stream().map(LoadedPlugin::instance).map(function);
```

Comment:
style nit: why add the `this`? it should be assumed for any method calls?

Reply:
Not sure how that got in there... but I'll take it out.